nomad/serf_test.go

```diff
@@ -88,6 +90,94 @@ func TestNomad_RemovePeer(t *testing.T) {
 	})
 }
 
+func TestNomad_ReapPeer(t *testing.T) {
```

Comment:
[nitpick] for lengthy tests I like a docstring describing what's being asserted.